### PR TITLE
Set-up-my-portfolio-part-1

### PR DESCRIPTION
**Pull request by Donard97**
This is my first real project and it is my portfolio. I had the template ready so I just needed to follow the instructions. 
The general requirements that I had to do are : 

1. Make sure that there are no linter errors
2. Make sure that you used the correct GitHub Flow
3. Make sure that you documented your work in a professional way
 
Project requirements are :

1. I had to build 2 sections: the header and the headline section.

I had to use flexbox on two sections.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  margin-top: 54px;
+}
+
+header nav img {
+  margin-left: 210px;
+  cursor: pointer;
+  color: #28352f;
+}
+
+/* First section style */
```
Comment (rloterh) [inline css/style.css]: - [ ] **OPTIONAL**
You left comments in your code hosted on the repository. To ensure cleaner code on your repository kindly remove all comments in your code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+          <h3>Welcome</h3>
```
Comment (rloterh) [inline index.html]: - [ ] Your logo is supposed to also serve as a link to your homepage. Kindly enwrap your logo with an `a` tag and provide the appropriate link.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          <img class="menu-icon" src="images/Union.png" alt="menu" />
+        </nav>
+      </header>
+
+      <!--First content-->
+    <section>
```
Comment (rloterh) [inline index.html:18]: - [ ] Kindly ensure that the content of this section is aligned with a `flexbox` as specified.
- [ ]  The contents of this section appears not centered. We actually want our design to mimic the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) as much as possible. Kindly ensure that the contents of this section are centered using appropriate margins and/or paddings.

**Your design:**
image


**The original design:** 
image

File: README.md [hunk 4]
```diff
@@ -1 +1,31 @@
 # Personal-Portofolio
+
+![image](https://user-images.githubusercontent.com/74506933/124952226-55e57180-e014-11eb-9970-21993938129c.png)
+
+This is my first website portofolio that I created for myself.
+
+# BUILT WITH
```
Comment (rloterh) [inline README.md:7]: - [ ] The naming of the headings in your ReadMe is not consistent. They are appearing in different letter casing. Kindly ensure to use a consistent letter casing for the headings in your ReadMe, preferably, sentence case, with the first letters of words capitalized.

File: README.md [hunk 5]
```diff
@@ -1 +1,31 @@
...
+HTML
+CSS
+
+# LIVE DEMO
+http://127.0.0.1:5500/index.html
+
+# AUTHOR
```
Comment (rloterh) [inline README.md:14]: - [ ] Kindly ensure to put your full name just after the Author heading, like as can be seen below:
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+          <li>
+            <a href="#"><img src="images/angellist.png" alt="angellist" /></a>
+          </li>
+        </ul>
+      </section>
+      <button id="btn" type="button">
+        <img src="images/Disabled.png" alt="button" />
+      </button>
+  
+
```
Comment (rloterh) [inline index.html]: - You left an extra empty line in your code that is serving no purpose. To ensure a cleaner code let's remove the extra empty line.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Donard , 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Descriptive comment messages
- No linter errors

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @Donard97,** 

### **Project Approved✅✅✅**
 Nice work on your portfolio, I wish you good luck in the next project 💪💪💪

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**